boards/arm/beagle_bbai_64/doc/index.rst

```diff
@@ -0,0 +1,91 @@
+.. _beaglebone_ai64:
```

Comment:
somewhere in your documentation please mention that the uart logs that you have enabled will appear on which port exactly on the BBAI-64?

Reply:
Adding uart 2 as console information below steps.